.eslintrc.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Let's use `.js` extension. It's much more flexible than JSON and we can add comments.

Reply:
ok